plugins/sentry/src/components/SentryPluginPage/SentryPluginPage.tsx

```diff
@@ -22,14 +22,17 @@ import {
   Content,
   ContentHeader,
   SupportButton,
+  useApi,
+  configApiRef,
 } from '@backstage/core';
 import { SentryPluginWidget } from '../SentryPluginWidget/SentryPluginWidget';
 import { ToggleButton, ToggleButtonGroup } from '@material-ui/lab';
 
 const SentryPluginPage: FC<{}> = () => {
   const [statsFor, setStatsFor] = useState<'12h' | '24h'>('12h');
   const toggleStatsFor = () => setStatsFor(statsFor === '12h' ? '12h' : '24h');
-  const sentryProjectId = 'sample-sentry-project-id';
+  const configApi = useApi(configApiRef);
+  const sentryProjectId = configApi.getString('sentry.project.id');
```

Comment:
Wondering how useful this actually is. Usually you'd have plenty more than one Sentry project/org right, most likely one per component?

Wondering if we should define an annotation for it instead: https://backstage.io/docs/features/software-catalog/well-known-annotations